source/graphql/migrate-hasura.txt

```diff
@@ -229,30 +149,25 @@ Cognito, or even a custom solution, to verify the user and set the necessary
 session variables. For configuring JWT or webhook authentication in Hasura,
 refer to the documentation at:
 
-- `JWT Authentication <https://hasura.io/docs/latest/auth/authentication/jwt/>`__
-- `Webhook Authentication <https://hasura.io/docs/latest/auth/authentication/webhook/>`__
+- `JWT Authentication <https://hasura.io/docs/3.0/auth/authentication/jwt/>`__
+- `Webhook Authentication <https://hasura.io/docs/3.0/auth/authentication/webhook/>`__
 
 .. _hasura-resolvers:
 
-Set Up Custom Resolvers
------------------------
+Set Up Custom Resolvers or Business Logic
+-----------------------------------------
 
 If your existing GraphQL API endpoints include custom resolvers or business
-logic, you'll need to implement these in Hasura. Hasura supports custom business
-logic using remote schemas, event triggers, and actions.
-
-Hasura Actions can be used to create custom resolvers. Custom queries and
-mutations including their payload type and input type can be defined within
-Hasura, and Hasura can integrate with the resolver function over HTTP. 
-
-.. image:: /images/migration/actions.png
-   :alt: Use Hasura Actions to create custom resolvers.
+logic, you'll need to implement these in Hasura. Hasura supports:
 
-The resolver function must be deployed somewhere and Hasura can generate the
-JavaScript code for the function.
-
-.. image:: /images/migration/resolver-codegen.png
-   :alt: Hasura auto generates Action code.
+- `Remote schemas <https://hasura.io/connectors/graphql>`__ through our GraphQL
```

Comment:
The "our" is confusing here. This documentation page is in the App Services docs set, so I would assume "our" refers to _us_ - but I don't think that's true. If the "our" here is intended to refer to Hasura, maybe we reword?

```suggestion
- `Remote schemas <https://hasura.io/connectors/graphql>`__ through its GraphQL
```